int32 id
geometry_msgs/Point32 vel